DirectEmployers/MyJobs
# Don't do this [PD-3461]

## What's this PR do?
Prevents us from overwriting a user's roles for another company when editing them.
## Screenshots (if appropriate)

## Where should the reviewer start?

## How should this be manually tested?
Make a user. Add a role to them with multiple companies. Edit that user through the user management interface, modifying their roles. Verify that that user still has roles with the other company.
## Any background context you want to provide?
